modular-docs-manual/content/topics/module_defining-user-stories.adoc

```diff
@@ -0,0 +1,77 @@
+[[defining-user-stories]]
+= Defining User Stories for Your Product
+
+User stories provide the context and structure from which you can determine which assemblies and modules to create.
+
+Ideally, well-defined user stories would already exist for the product you are documenting. For most writers, however, this ideal is not a reality. If you do not have any user stories from which to work, and you -- as a writer -- do not have all of the user information you would need to create the user stories yourself, how do you get started? This procedure provides a general approach that you can take. Of course, every product differs in terms of tools, processes, team dynamics, and access to SMEs. Since you are most familiar with these aspects for your own team, you will need to adapt this general approach for your own team.
+
+.Procedure
+
+. Collect any existing user stories for your product.
++
+You might need to talk to your Product Manager and Content Strategist.
+
+. Identify the key, top-level user stories for your product.
++
+--
+If your product does not have any user stories, you can use the customer product lifecycle to get started. You can consider each phase of the lifecycle to be a user story.
+
+You may not need to use every phase of the customer lifecycle, but for now, you should pick the ones that you know apply to your product. In the next step, you will start to refine them.
+
+For more information, see xref:customer-product-lifecycle[Overview of the Customer Product Lifecycle].
+--
+
+. Define the supporting user stories that are necessary to complete each of the top-level user stories.
++
+--
+Each top-level user story represents a "phase" of the customer lifecycle. You should go through each phase and define the user stories needed to complete the phase. You will need to use your own knowledge and expertise of the product.
+
+For example, for the _Configure and Verify_ phase, users would need to know how to configure and set up each component or feature of the product to work in their environment. You could break it down like this:
+
+.Creating Second-Level User Stories
+====
+* Configure Product X so that it is ready for production
```

Comment:
I'd recommend to use the full format of a user story in the workflow doc -- including the "As a [user]" and "so that [reason]" bits whenever possible. Not because I'd enjoy being pedantic, but because the complete user story gives the necessary context, for example about who the user is and what their level of knowledge is. Consider the difference between "As a sysadmin/As a manager/As an end user/As a security officer/..." -- this all could influence what the corresponding assembly should cover.

Reply:
Done.

I'm glad you pointed this out, because it will at least force everyone to consider audience groups. In fact, we're probably missing a Content Strategist step to define the product audiences. Each audience might have different or even overlapping user stories for each lifecycle phase.